packages/@expo/cli/src/api/user/__tests__/actions-test.ts

```diff
@@ -14,7 +14,10 @@ jest.mock('../../rest/client', () => {
   };
 });
 jest.mock('../otp');
-jest.mock('../user');
+jest.mock('../user', () => ({
+  loginAsync: jest.fn(),
+  ssoLoginAsync: jest.fn(),
+}));
```

Comment:
Move these to the shared mock since the other tests in this file presumably expect the additional mocks https://github.com/expo/expo/blob/main/packages/%40expo/cli/src/api/user/__mocks__/user.ts

Reply:
Done